### PR TITLE
add packer-plugin-amazon package

### PR DESCRIPTION
**Pull request by kranurag7**
no tests given we don't have packer

### CODE REVIEW

File: packer-plugin-amazon.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  - uses: go/bump
+    with:
+      deps: github.com/go-jose/go-jose/v3@v3.0.3 github.com/hashicorp/go-retryablehttp@v0.7.7 golang.org/x/crypto@v0.17.0 golang.org/x/net@v0.23.0 google.golang.org/protobuf@v1.33.0 github.com/hashicorp/packer-plugin-sdk@v0.5.4
+      replaces: github.com/go-jose/go-jose/v3=github.com/go-jose/go-jose/v3@v3.0.3 gopkg.in/square/go-jose.v2=github.com/go-jose/go-jose/v3@v3.0.3
+
+  - uses: go/build
+    with:
+      packages: .
+      output: packer-plugin-amazon
+
```
Comment (mamccorm) [inline packer-plugin-amazon.yaml:24]: Does the binary respond to a --version or --help? Given it's a go binary it's highly likely, and we could at least add a basic test

Comment (mamccorm) [reply packer-plugin-amazon.yaml:24]: Hi @ kranurag7, is this PR still needed?

Comment (mamccorm) [reply packer-plugin-amazon.yaml:24]: Have we been able to do any manual validation that the plugins are detected and usable by packer?

**Comment by kranurag7** [comment]
closing this for now given this requires more work and we don't have confirmation yet on the tag.

**Pull request closed without merging**